tests/secadvisor_test.go

```diff
@@ -0,0 +1,188 @@
+// +build sa
```

Comment:
Please use `sa_tests` instead

Reply:
renamed

Reply:
Sorry, my comment was not clear. I was taking about the build tag, not the file name